utils/dataloaders.py

```diff
@@ -13,7 +16,7 @@ def get_img_size(dataset):
     """Return the correct image size."""
     if dataset in ['mnist', 'fashion_mnist']:
         img_size = (1, 32, 32)
-    if dataset in ['chairs', 'dsprites']:
+    if dataset in ['3DChairs', 'dsprites']:
```

Comment:
if you change here, you have to change the main.py ! did you actually test from main.py ? pretty sure that would not work!